ks/azure/centos76-hpc-sriov.ks

```diff
@@ -0,0 +1,401 @@
+# Kickstart for provisioning a CentOS 7.6 Azure HPC (SR-IOV) VM
+
+# System authorization information
+auth --enableshadow --passalgo=sha512
+
+# Use graphical install
+text
+
+# Do not run the Setup Agent on first boot
+firstboot --disable
+
+# Keyboard layouts
+keyboard --vckeymap=us --xlayouts='us'
+
+# System language
+lang en_US.UTF-8
+
+# Network information
+network --bootproto=dhcp
+
+# Use network installation
+url --url="http://olcentgbl.trafficmanager.net/centos/7.6.1810/os/x86_64/"
+repo --name "os" --baseurl="http://olcentgbl.trafficmanager.net/centos/7.6.1810/os/x86_64/" --cost=100
+repo --name="updates" --baseurl="http://olcentgbl.trafficmanager.net/centos/7.6.1810/updates/x86_64/" --cost=100
+repo --name "extras" --baseurl="http://olcentgbl.trafficmanager.net/centos/7.6.1810/extras/x86_64/" --cost=100
+repo --name="openlogic" --baseurl="http://olcentgbl.trafficmanager.net/openlogic/7/openlogic/x86_64/"
+
+# Root password
+rootpw --plaintext "to_be_disabled"
+
+# System services
+services --enabled="sshd,waagent,dnsmasq,NetworkManager"
+
+# System timezone
+timezone Etc/UTC --isUtc
+
+# Partition clearing information
+clearpart --all --initlabel
+
+# Clear the MBR
+zerombr
+
+# Disk partitioning information
+part /boot --fstype="xfs" --size=500
+part / --fstype="xfs" --size=1 --grow --asprimary
+
+# System bootloader configuration
+bootloader --location=mbr --timeout=1
+
+# Firewall configuration
+firewall --disabled
+
+# Enable SELinux
+selinux --enforcing
+
+# Don't configure X
+skipx
+
+# Power down the machine after install
+poweroff
+
+# Disable kdump
+%addon com_redhat_kdump --disable
+%end
+
+%packages
+@base
+@console-internet
+@development-tools
+ntp
+cifs-utils
+sudo
+python-pyasn1
+parted
+WALinuxAgent
```

Comment:
Should we not be installing this since you are installing manually below?  Hopefully we can re-add this once the latest agent package is in the repo.

Reply:
Thanks, yes, we don't need it here.